src/sentry/static/sentry/app/views/organizationDiscover/utils.jsx

```diff
@@ -0,0 +1,30 @@
+export function getQueryFromQueryString(queryString) {
+  if (!queryString) {
+    return {};
+  }
+  let parsedQuery = queryString;
```

Comment:
can we use const wherever possible, since we'll be doing that throughout our codebase shortly?

Reply:
right but i can't change parsedQuery to const since I'm using regex on it later.

Reply:
gotcha, put this on the wrong line, just the other instances below